shipit.yml

```diff
@@ -16,3 +16,5 @@ merge:
 
 deploy:
   max_commits: false
+  override:
+    - npm publish
```

Comment:
This ensures `publishConfig.tag` is respected - because Shipit wouldn't have done so ([See issue in shipit-engine](https://github.com/Shopify/shipit-engine/issues/843)).